site/_resources/release-process.md

```diff
@@ -175,7 +184,7 @@ Set environment variables for use in subsequent steps:
 export CONTOUR_RELEASE_VERSION=v1.11.1
 export CONTOUR_RELEASE_VERSION_MAJOR=1
 export CONTOUR_RELEASE_VERSION_MINOR=11
-export CONTOUR_PREVIOUS_VERSION=v1.11.0
+export CONTOUR_PREVIOUS_VERSION=v1.10.0
```

Comment:
I think this env var needs to stay as `v1.11.0` for the purposes of docs/YAML generation, so probably add a new one for the purposes of upgrade testing from the previous minor release.

Reply:
oh good call, yeah totally spaced that was 1.11.0 for the bump to 1.11.1